tests/test-rest-comments-controller.php

```diff
@@ -21,6 +21,9 @@ class WP_Test_REST_Comments_Controller extends WP_Test_REST_Controller_Testcase
 	public function setUp() {
 		parent::setUp();
 
+		// Don't fail on flood.
+		remove_filter( 'check_comment_flood', 'check_comment_flood_db' );
```

Comment:
> This shouldn't be added as a part of this pull request.

To clarify, I don't think this is the appropriate fix. We should likely resolve #1660, #790